library/xwiki

```diff
@@ -1,21 +1,37 @@
 Maintainers: XWiki Core Dev Team <committers@xwiki.org> (@xwiki)
 GitRepo: https://github.com/xwiki-contrib/docker-xwiki.git
 
-# Stable
-Tags: 17, 17.10, 17.10.0, 17-mysql-tomcat, 17.10-mysql-tomcat, 17.10.0-mysql-tomcat, mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest
+# LTS & Stable (until 18.0.0RC1)
+Tags: 17, 17.10, 17.10.2, 17-mysql-tomcat, 17.10-mysql-tomcat, 17.10.2-mysql-tomcat, mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest, lts-mysql-tomcat, lts-mysql, lts
 Architectures: amd64, arm64v8
 Directory: 17/mysql-tomcat
-GitCommit: b1dc0934aa1761ef397459f7dbf866480e9fc6c9
+GitCommit: a7602018132edae5f6fc023fd172692ff2794452
 
-Tags: 17-postgres-tomcat, 17.10-postgres-tomcat, 17.10.0-postgres-tomcat, postgres-tomcat, stable-postgres-tomcat, stable-postgres
+Tags: 17-postgres-tomcat, 17.10-postgres-tomcat, 17.10.0-postgres-tomcat, postgres-tomcat, stable-postgres-tomcat, stable-postgres, lts-postgres-tomcat, lts-postgres
```

Comment:
You might want to look at writing this file's contents with a script 😅  (that's what we do, and I'm happy to help with that if you need it)

there's still a 17.10.0 here that I think you meant to update to 17.10.2 👀